docs/debug.md

```diff
@@ -33,7 +33,7 @@ You can customize the core dump pattern with the following options
 
 There is a debug snap that can be installed with:
 
-    $ sudo snappy install snappy-debug
+    $ sudo snap install snappy-debug
```

Comment:
Yeah, but it may not be in the form of a snap; so this should ideally be removed, it is a VCS after all and we can recover it ;-)